Project/MyGameLibrary/InventoryItems/HealthPotion.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fall2020_CSC403_Project.code;
+
+namespace MyGameLibrary.InventoryObjects
+{
+    public class HealthPotion : InventoryObject
+    {
+        private readonly int healAmount = 6;
+        
+        public HealthPotion(Image img)
+        {
+            this.stackable = false;
+            this.exhaustible = true;
+            this.img = img;
+            
+        }
+
+        public override void Effect(BattleCharacter character)
+        {
+            throw new NotImplementedException();
+
```

Comment:
Extra new line added here, there are several places in other files as well. After making the other changes from this review, run through the code and remove them